.gitignore

```
@@ -84,3 +74,14 @@ artifacts
 packages/prepo-sdk/lib
 tmp
 docs
+
+# --------- prepo-core ---------
+typechain/
+apps/core/artifacts*
+apps/core/cache*
+apps/core/cache
+cache
+apps/core/deployments/localhost
+apps/core/deployments/goerli
+apps/core/.openzeppelin/unknown-*.json
```

```suggestion
apps/core/.openzeppelin/unknown-31337.json
```
This is something I realized later on that Polygon deployments also fall under "unknown" and we don't want to ignore those. So instead we should specify exactly the `hardhat` test chainId 31337.